### ISSUE TITLE
Please specify delay/ lag for when logs are available in API/ through PowerShell

### ISSUE THREAD

**Issue by o-l-a-v** [MicrosoftDocs/powerbi-docs#2634]
## Background

According to following documentation, there is a lag of 30 minutes before one can expect all logs to appear in the Power BI audit log:

* https://docs.microsoft.com/en-us/power-bi/guidance/admin-activity-log#powershell-sample
![image](https://user-images.githubusercontent.com/6450056/99183848-43de5800-273f-11eb-9482-4be3ae6f2872.png)

According to my own experience with the ```Search-UnifiedAuditLog``` one can indeed experience lag/ delay, when it comes to when logs/ events are made available for API/ PowerShell.

API documentation says nothing about it:

* https://docs.microsoft.com/en-us/rest/api/power-bi/admin/getactivityevents

This documentation says nothing about it:

* https://docs.microsoft.com/en-us/power-bi/admin/service-admin-auditing

```Search-UnifiedAuditLog``` says nothing about it:

* https://docs.microsoft.com/en-us/powershell/module/exchange/search-unifiedauditlog

## Question/ request

Can you please specify or refer to a source where this delay is explained in details?

* Is it the same for the API, Exchange Online ```Search-UnifiedAuditLog``` and Power BI module ```Get-PowerBIActivityEvent```?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a0e8bce0-95c1-07f7-d3a3-a1c22bd21f29
* Version Independent ID: dddd1e30-4a88-1293-1264-3214cc4b6dc8
* Content: [Track user activities in Power BI - Power BI](https://docs.microsoft.com/en-us/power-bi/admin/service-admin-auditing)
* Content Source: [powerbi-docs/admin/service-admin-auditing.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/master/powerbi-docs/admin/service-admin-auditing.md)
* Service: **powerbi**
* Sub-service: **powerbi-admin**
* GitHub Login: @kfollis
* Microsoft Alias: **kfollis**

**Comment by paulinbar** [comment]
Hi @o-l-a-v , thanks for reaching out to us with your comment. I have forwarded it on to the doc author for further consideration.

**Comment by kfollis** [comment]
@o-l-a-v - I apologize that this feedback has been idle. We have some staff changes for activity and audit logs, and I am passing this feedback on to them to ensure we address this lack of detail.

**Comment by kfollis** [comment]
I am closing this issue now as the admin auditing article has been updated to reflect this time delay. A reference to all services found in M365 compliance can be found here: https://docs.microsoft.com/en-us/microsoft-365/compliance/search-the-audit-log-in-security-and-compliance?view=o365-worldwide#before-you-search-the-audit-log. The other articles you mentioned are automated reference, however, I have contacted the article owners to recommend that a note about the delay be manually added.
